internal/db/sqltest/tests/session/session_connection_state_transition.sql

```diff
@@ -0,0 +1,35 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+    select plan(6);
+
+    -- Ensure session connection table is populated
+    select is(count(*), 2::bigint) from session_connection;
+
+    -- Check that both session connections are in the authorized state (null connected_time_range)
+    select is(count(*), 2::bigint) from session_connection where connected_time_range is null;
+
+    -- Connect one of the session connections
+    update session_connection
+        set connected_time_range=tstzrange(now(),'infinity')
+        where public_id = 's1c1___clare';
```

Comment:
Likewise, can you format blocks like these to form a river?:


```suggestion
    update session_connection
       set connected_time_range = tstzrange(now(), 'infinity')
     where public_id = 's1c1___clare';
```